web3/providers/persistent.py

```diff
@@ -34,11 +34,13 @@ class PersistentConnectionProvider(AsyncJSONBaseProvider, ABC):
     def __init__(
         self,
         request_timeout: float = DEFAULT_PERSISTENT_CONNECTION_TIMEOUT,
+        request_information_cache_size: int = 500,
```

Comment:
It probably doesn't matter much, but the order of the kwargs may be confusing if anyone was using the `subscription_response_queue_size`. Consider adding it after `subscription_response_queue_size`?